group/assignment5/Vertex.java

```diff
@@ -0,0 +1,55 @@
+package assignment5;
+
+import java.util.HashSet;
+
+public class Vertex {
+
+    private Character letter;

```

Comment:
You could make letter final since the character of a vertex can't change. That's something I usually forget.